src/llmq/quorums.cpp

```diff
@@ -1100,13 +1103,75 @@ void CQuorumManager::StartCleanupOldQuorumDataThread(const CBlockIndex* pIndex)
         }
 
         if (!quorumThreadInterrupt) {
-            DataCleanupHelper(m_evoDb.GetRawDB(), dbKeysToSkip);
+            WITH_LOCK(cs_db, DataCleanupHelper(*db, dbKeysToSkip));
         }
 
         LogPrint(BCLog::LLMQ, "CQuorumManager::StartCleanupOldQuorumDataThread -- done. time=%d\n", t.count());
     });
 }
 
+void CQuorumManager::MigrateOldQuorumDB(CEvoDB& evoDb) const
```

Comment:
Maybe it worth to add TODO here to remove it in the future with expected timeline?